src/main/kotlin/com/target/slack/EmojiService.kt

```diff
@@ -609,21 +663,23 @@ class EmojiService(private val config: Config, private val db: Db) {
      *
      */
     fun tallyVotes(prop: Proposal): TallyResult {
-        var tallyResult = TallyResult(0, 0, 0, 0)
+        var tallyResult = TallyResult(0, 0, 0, 0, 0, 0)
```

Comment:
this would benefit from using named args assuming TallyResult is a kotlin class. otherwise it's really hard to know what those args are, especially since they're all the same type.

Reply:
Its a data class (defined at the top of this same file).  Probably makes sense to set the default values for each to be 0 so you can call it with with an empty constructor.